htdocs/core/boxes/box_factures.php

```diff
@@ -93,7 +93,8 @@ public function loadBox($max = 5)
 			$sql .= ", f.tva as total_tva";
 			$sql .= ", f.total_ttc";
 			$sql .= ", f.datef as df";
-			$sql .= ", f.paye, f.fk_statut as status, f.datec, f.tms";
+			$sql .= ", f.paye, f.fk_statut, f.datec, f.tms";
```

Comment:
Warning, during your merge i think you lost a change done by another PR. For example
f.fk_statut was replaced with f.fk_statut as status 
this change must remain in your own PR.